From linux-usb-owner@vger.kernel.org Mon Dec  3 14:15:59 2007
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Mon, 3 Dec 2007 14:14:16 -0800
Subject: usb.h: fix kernel-doc warning
To: linux-usb@vger.kernel.org
Cc: gregkh <greg@kroah.com>
Message-ID: <20071203141416.10126e12.randy.dunlap@oracle.com>


From: Randy Dunlap <randy.dunlap@oracle.com>

Fix kernel-doc warning in usb.h:
Warning(linux-2.6.24-rc3-git7//include/linux/usb.h:166): No description found for parameter 'sysfs_files_created'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/usb.h |    1 +
 1 file changed, 1 insertion(+)

--- a/include/linux/usb.h
+++ b/include/linux/usb.h
@@ -107,6 +107,7 @@ enum usb_interface_condition {
  * @condition: binding state of the interface: not bound, binding
  *	(in probe()), bound to a driver, or unbinding (in disconnect())
  * @is_active: flag set when the interface is bound and not suspended.
+ * @sysfs_files_created: sysfs attributes exist
  * @needs_remote_wakeup: flag set when the driver requires remote-wakeup
  *	capability during autosuspend.
  * @dev: driver model's view of this device
